### ISSUE TITLE
Rework the API Documentation so that it's more clear.

### ISSUE THREAD

**Issue by ncb000gt** [kelektiv/node-cron#441]
The API documentation hasn't changed much since I threw it together with the original module. Clearer documentation has always been a source of irritation for users.

**Comment by Quadriphobs1** [comment]
Would love to create PR for this

**Comment by ncb000gt** [comment]
That would be great. Thanks!

**Comment by balajipachai** [comment]
One quick question: reworking the API documentation does it imply, changing it in the API section/ adding code examples in the README section?

**Comment by ncb000gt** [comment]
@balajipachai yes. Right now it's too condensed and kind of hard to read unless you know it well. The readme itself isn't broken out well.

An example of something I'm looking for is something like: https://github.com/RobinBuschmann/sequelize-typescript

**Comment by keplerk** [comment]
Indeed that would be nice, thumbs up for this

**Comment by nmay231** [comment]
Another thing that could be done is to host the example docs on something like github.io or gitbook (premium is free for open-source projects) and then get a domain from [js.org](https://github.com/js-org/js.org). Both `cron.js.org` and `node-cron.js.org` are available.

Don't know if it's something you want to do for this project, but it's an idea. :man_shrugging:

**Comment by ncb000gt** [comment]
This sounds like a great idea. :)

- Nick Campbell

**Comment by sheerlox** [comment]
some good project READMEs to use as inspiration:
- [axios/axios](https://github.com/axios/axios)
- [jestjs/jest](https://github.com/jestjs/jest)
- [sequelize/sequelize-typescript](https://github.com/sequelize/sequelize-typescript)
